switch_utils/switch_utils.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
```

Comment:
I would recommend including the SPDX License identifier (say, right under the copyright line), even if you're including the text of a license in its entirety.

The SPDX license identifier is designed to be unambiguous and easily parsed (e.g., by software). See [SPDX IDs](https://spdx.dev/ids/) for more information.